File: Using C Language/bmiCalculator.c
// Program to calculate BMI

#include<stdio.h>

void main()
{
    float height, weight, BMI;

    // Scanning the height from the user
    printf("Enter your height in metre: ");
    scanf("%f",&height);

    // Scanning the weight from the user
    printf("Enter your weight in kilogram: ");
    scanf("%f",&weight);

    // Calculating BMI
    BMI = weight / (height * height);

    // Printing BMI
    printf("\nYor BMI: %f",BMI);

    // Printing Conclusion of BMI
    if(BMI < 18.5)
    {
        printf("\nConclusion: UNDERWEIGHT\n");
    }
    else if(BMI > 18.5 && BMI < 24.9)
    {
        printf("\nConclusion: HEALTHY\n");
    }
    else if(BMI > 25.0 && BMI < 29.9)
    {
        printf("\nConclusion: OVERWEIGHT\n");
    }
    else
    {
        printf("\nConclusion: OBESE\n");
    }
}

File: Using Dart/bmiCalculator.dart
// Program to calculate BMI

import 'dart:io';

void main() {
  // Scanning the height from the user
  stdout.write("Enter your height in metre: ");
  double? height = double.parse(stdin.readLineSync()!);

  // Scanning the weight from the user
  stdout.write("Enter your weight in kilogram: ");
  double? weight = double.parse(stdin.readLineSync()!);

  // Calculating BMI
  double? bmi = weight / (height * height);

  // Printing the BMI
  print("Your BMI: $bmi");

  // Printing the conclusion
  if (bmi < 18.5) {
    print("Conclusion: UNDERWEIGHT");
  } else if (bmi > 18.5 && bmi < 24.9) {
    print("Conclusion: HEALTHY");
  } else if (bmi > 25.0 && bmi < 29.9) {
    print("Conclusion: OVERWEIGHT");
  } else {
    print("Conlusion: OBESE");
  }
}
